releasenotes/notes/add-serializable-parametric-pulse-31490c4d2cc49ec6.yaml

```diff
@@ -0,0 +1,26 @@
+---
+features:
+  - |
+    New pulse base class :class:`~SymbolicPulse` has been added. This is a
+    replacement of the conventional :class:`~ParametricPulse` which will be deprecated.
+    In new base class, pulse envelope and parameter validation functions are
+    represented by symbolic expression objects, which are lambidfied immediately when
+    corresponding methods are called for the first time.
+    The new class provides a self-contained and portable pulse data since these symbolic equations
+    can be easily serialized through symbolic computation libraries.
+    QPY serialization support of these pulses is planned.
+upgrade:
+  - |
+    Importing pulse class from ``qiskit.pulse.library.*`` now import :class:`~SymbolicPulse`
+    subclass rather than :class:`~ParametricPulse` subclass.
+    There are same set of pulse subclass in the symbolic pulse module,
+    :class:`~qiskit.pulse.library.Gaussian`,
+    :class:`~qiskit.pulse.library.GaussianSquare`,
+    :class:`~qiskit.pulse.library.Drag`, and
+    :class:`~qiskit.pulse.library.Constant`.
+    Becasue :class:`~SymbolicPulse` supports full backward compatibility,
+    there is no loss of functionality due to the switching of the base class.
+deprecations:
+  - |
+    A pulse base class :class:`~ParametricPulse` has been deprecated and replaced with
+    :class:`~SymbolicPulse`.
```

Comment:
We should remove this because we haven't actually deprecated `ParametericPulse` yet it's a pending deprecation. We can do this in 0.22.0

```suggestion
```